### PR TITLE
Setting up keyvault live test

### PR DESCRIPTION
**Pull request by vhvb1989**
Setting up KeyVault live testing

### CODE REVIEW

File: sdk/keyvault/test-resources.json [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "tenantId": {
+            "type": "string",
+            "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```
Comment (RickWinter) [inline sdk/keyvault/test-resources.json:14]: Why is this the default vs populating the value via parameter?

File: sdk/keyvault/test-resources.json [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            "metadata": {
+                "description": "The tenant ID to which the application and resources belong."
+            }
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
```
Comment (RickWinter) [inline sdk/keyvault/test-resources.json:21]: Ditto on the default value.

**Review by RickWinter — APPROVED** [review]

File: sdk/keyvault/ci.yml [hunk 3]
```diff
@@ -34,6 +34,8 @@ stages:
     parameters:
       ServiceDirectory: keyvault
       CtestRegex: azure-security-keyvault
+      LiveTestCtestRegex: live-azure-security-keyvault
+      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
```
Comment (danieljurek) [inline sdk/keyvault/ci.yml:38]: Do we need to use the canary subscription specifically or can we use the normal subscription `$(sub-config-azure-cloud-test-resources)` ?

Comment (vhvb1989) [reply sdk/keyvault/ci.yml:38]: I copied from storage.  I don't really know what should I use :(

Comment (danieljurek) [reply sdk/keyvault/ci.yml:38]: By default use `$(sub-config-azure-cloud-test-resources)` as it is our normal subscription. Canary and other subscriptions have different features enabled but `$(sub-config-azure-cloud-test-resources)` is a more pristine subscription.

Comment (weshaggard) [reply sdk/keyvault/ci.yml:38]: That is true assuming the canary features don't require any feature flags. I believe storage in particular needed feature flags so we kept them in the preview subscription where we are OK with feature flags.

**Pull request merged**